web/templates/user_profile_modal.hbs

```diff
@@ -18,16 +18,18 @@
                         <i class="zulip-icon zulip-icon-bot" aria-hidden="true"></i>
                     {{/if}}
                     <span class="user-profile-name">{{> user_full_name name=full_name}}</span>
-                    <a id="user_profile_manage_own_copy_link" class="copy-link-to-user-profile" tabindex="0" data-user-id="{{user_id}}">
-                        <i class="fa fa-link tippy-zulip-tooltip user-profile-manage-own-copy-link-button" data-tippy-content="{{t 'Copy link to profile' }}" aria-hidden="true"></i>
+                    <a class="copy-link-to-user-profile user-profile-manage-own-copy-link-button" tabindex="0" data-user-id="{{user_id}}">
+                        <i class="zulip-icon zulip-icon-link tippy-zulip-tooltip " data-tippy-content="{{t 'Copy link to profile' }}" aria-hidden="true"></i>
```

Comment:
The commit message doesn't say anything about moving this class to the containing `a` element while you're at it.